src/script.js

```diff
@@ -163,53 +163,41 @@ function getUserInputValues() {
         // Pridobi vnose energij
```

Comment:
I would try to avoid "||" a lot of times is problamatic use rather if else or tenries ... you can do  something like 

for (let i = 1; i < 6; i++) {
    // Pridobi vnose energij
    let e_cbx = parseFloat(document.getElementById(`e-cb${i}`).value);
    e_cb.push(!isNaN(e_cbx) && e_cbx >= 0 ? e_cbx : 0);
    
    // Pridobi vnose dog. moči
    let m_cbx = parseFloat(document.getElementById(`m-cb${i}`).value);
    m_cb.push(!isNaN(m_cbx) && m_cbx >= 0 ? m_cbx : 0);
    
    // Pridobi vnose presežkov
    let m_ex_cbx = parseFloat(document.getElementById(`m-ex-cb${i}`).value);
    m_ex_cb.push(!isNaN(m_ex_cbx) && m_ex_cbx >= 0 ? m_ex_cbx : 0);
}